### PR TITLE
added alternative solution for Longest word problem

### PR DESCRIPTION
**Pull request by sushilkundu143**
This is a alternative solution for finding the longest word form a sentence.

### CODE REVIEW

**Review by ZLester — CHANGES_REQUESTED** [review]
Please fix your variable spacing and change the comment to `// Using Array sort`

**Review by sushilkundu143 — COMMENTED** [review]
- 'Using Array sort method' comment added.
- Varible spacing fixed. 

Please review the code and merge.

File: Easy/03 - Longest Word.js [hunk 1]
```diff
@@ -47,3 +47,13 @@ function LongestWord(sen) {
...
 }
+
+
+// alternative solution of longest word problem (Using Array sort method)
+function LongestWord(sen) { 
+    var words = sen.match(/[a-z0-9]+/gi);
+    var sorted = words.sort(function(a,b){
+       return b.length - a.length; 
+    });
+     return sorted[0];
```
Comment (ZLester) [inline Easy/03 - Longest Word.js]: Extra space here

File: Easy/03 - Longest Word.js [hunk 2]
```diff
@@ -47,3 +47,13 @@ function LongestWord(sen) {
     }
     return longestWord;
 }
+
+
+// alternative solution of longest word problem (Using Array sort method)
+function LongestWord(sen) { 
+    var words = sen.match(/[a-z0-9]+/gi);
+    var sorted = words.sort(function(a,b){
```
Comment (ZLester) [inline Easy/03 - Longest Word.js]: Please have your function braces consistently spaced

File: Easy/03 - Longest Word.js [hunk 3]
```diff
@@ -47,3 +47,13 @@ function LongestWord(sen) {
     }
     return longestWord;
 }
+
+
+// alternative solution of longest word problem (Using Array sort method)
```
Comment (ZLester) [inline Easy/03 - Longest Word.js]: Please change to `// Using Array sort Method`

Comment (sushilkundu143) [reply Easy/03 - Longest Word.js]: Just pushed by resolving the spacing issue. Please review the code.

**Review by sushilkundu143 — COMMENTED** [review]
Pushed after resolving the spacing issue. Please review it.

**Comment by ZLester** [comment]
Please update your comment to what I requested and add a space before the function opening brace in your .sort callback per my review.

**Review by sushilkundu143 — COMMENTED** [review]
Comment added . Please review the code.

File: Easy/03 - Longest Word.js [hunk 4]
```diff
@@ -47,3 +47,13 @@ function LongestWord(sen) {
     }
     return longestWord;
 }
+
+
+// Using Array sort() Method
+function LongestWord(sen) { 
+    var words = sen.match(/[a-z0-9]+/gi);
+    var sorted = words.sort(function(a,b){
```
Comment (ZLester) [inline Easy/03 - Longest Word.js:55]: Still missing a space before your function brace, but I'll add eslint to the repo so things like this get marked automatically.

**Review by ZLester — APPROVED** [review]

**Pull request merged**